src/components/Sum.jsx

```diff
@@ -1 +1,16 @@
-// export function Sum() {}
+import React from 'react';
+
+function Sum(props) {
```

Comment:
```suggestion
export function Sum(props) {
```

you may do like this, to not `export { Sum };` in the end of the file 😉